interactions/models/internal/context.py

```diff
@@ -598,7 +598,9 @@ async def defer(self, *, ephemeral: bool = False, edit_origin: bool = False) ->
             raise RuntimeError("Interaction has already been responded to.")
 
         payload = {
-            "type": CallbackType.DEFERRED_CHANNEL_MESSAGE_WITH_SOURCE if not edit_origin else CallbackType.DEFERRED_UPDATE_MESSAGE
+            "type": CallbackType.DEFERRED_CHANNEL_MESSAGE_WITH_SOURCE
```

Comment:
This does not appear related to your what you are trying to resolve - please rebase your pr before it can be merged

Reply:
This aint my changes, this has been automatically formatted by pre-commit-ci[

Reply:
Okeydoke, ill ignore it then